src/mapgen_functions.cpp

```diff
@@ -1357,6 +1356,7 @@ void mapgen_sewer_four_way( mapgendata &dat )
 }
 
 ///////////////////
+#if 0 // preserving for reference
```

Comment:
It's always preserved in the git history. It shouldn't be necessary to keep it here.

Reply:
Fixed, thanks.